lua/core/autopairs.lua

```diff
@@ -1,51 +1,57 @@
--- if not package.loaded['nvim-autopairs'] then
---   return
--- end
-local status_ok, _ = pcall(require, "nvim-autopairs")
-if not status_ok then
-  return
+local M = {}
+
+M.config = function()
+  lvim.builtin.autopairs = {
+    active = true,
+  }
 end
-local npairs = require "nvim-autopairs"
-local Rule = require "nvim-autopairs.rule"
 
--- skip it, if you use another global object
-_G.MUtils = {}
+M.setup = function()
+  local autopairs = require "nvim-autopairs"
+  local Rule = require "nvim-autopairs.rule"
+
+  -- skip it, if you use another global object
+  _G.MUtils = {}
 
-vim.g.completion_confirm_key = ""
-MUtils.completion_confirm = function()
-  if vim.fn.pumvisible() ~= 0 then
```

Comment:
Just in case this gets merged and your other PR closed, this might be necessary

Reply:
I think it should be fine now